scripts/helpers.js

```diff
@@ -2,124 +2,119 @@
 
 'use strict';
 
-var pathFn = require('path');
-var _ = require('lodash');
-var cheerio = require('cheerio');
-var lunr = require('lunr');
+const pathFn = require('path');
```

Comment:
since this is not a browser script, I think it's safe to destructure.

Reply:
Updated.
Destructuring are not supported in Node 4.0.0 so it won't pass eslint. Pending #1138.